third_party/intel/backend/compiler.py

```diff
@@ -296,7 +300,60 @@ def make_spv(src, metadata, options):
         else:
             metadata["build_flags"] = ""
 
-        return ret
+        if options.generate_native_code is True:
+            with tempfile.NamedTemporaryFile(delete=False, mode='wb', suffix='.spv') as fsrc, \
+                tempfile.NamedTemporaryFile(delete=False, mode='r', suffix='.log') as flog:
+                fsrc.write(spirv)
+                fsrc.flush()
+                fbin = fsrc.name + '.o'
+
+                ocloc_cmd = [
+                    'ocloc', 'compile', '-file', fsrc.name, '-o', fbin, '-spirv_input', '-device', 'pvc', '-options',
+                    f'{metadata["build_flags"]}'
+                ]
+
+                try:
+                    subprocess.run(ocloc_cmd, check=True, close_fds=False, stdout=flog, stderr=subprocess.STDOUT)
```

Comment:
```suggestion
                    subprocess.run(ocloc_cmd, check=True, close_fds=False, stdout=flog, stderr=subprocess.STDOUT)
```

Why do we redirect `stderr` to `stdout`?

Reply:
It was not clear to me that you can capture both to the same file descriptor.